llama_index/ingestion/pipeline.py

```diff
@@ -120,6 +120,7 @@ class DocstoreStrategy(str, Enum):
 
     UPSERTS = "upserts"
     DUPLICATES_ONLY = "duplicates_only"
+    UPSERTS_AND_DELETE = "upserts and delete"
```

Comment:
nit: can we use underscores for enum strings?

Reply:
@logan-markewich , Done. Thanks!

Reply:
Thank you!